config/rbac/role.yaml

```diff
@@ -70,3 +70,10 @@ rules:
   verbs:
   - get
   - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  - pods
+  verbs:
+  - create
```

Comment:
Can we add `create` to nodes and pods resources earlier in the file on line # 60-61

Reply:
I had originally not wanted to allow create pods, but the more I think about it, we probably want to do this for sharding.

Reply:
Wait nevermind, we'd want create deployment, or something. Leaving as is.